index.js

```diff
@@ -91,6 +91,69 @@ say.speak = function(text, voice, speed, callback) {
   });
 };
 
+say.export = function(text, output, params, callback) {
+  var commands, pipedData;
+
+  if (!text) {
+    // throw TypeError because API was used incorrectly
+    throw new TypeError('Must provide text parameter');
+  }
+
+  if (!output || typeof output !== 'string') {
+    // throw TypeError because API was used incorrectly
+    throw new TypeError('Must provide output parameter');
+  }
+
+  if (typeof params !== 'object'){
+    params = {voice: 'Alex', speed: 175};
```

Comment:
If these params aren't passed in, simply default to nulls. That way, if `say` changes defaults in the future, we won't have incorrect values.